product_subscription/views/subscription_views.xml

```diff
@@ -171,6 +171,7 @@
                 <form string="Subscription Request">
 		            <header>
 		            	<button name="validate_request" string="Validate request" type="object" states="draft" class="btn-primary"/>
+		            	<button name="force_subscription" string="Force subscription creation" type="object" states="sent"/>
```

Comment:
tu mettrais pas un domain pour qu'on  ne voit le bouton que 
- quand l'`origin == 'manual'`
- ou quand `subscription_id is False`
?

Reply:
non car on doit pouvoir corriger des request créées sur le website dans le backend et valider par la suite. par contre le domaine ne passe pas lorsque subscription est défini